README.md

```diff
@@ -1,2 +1,54 @@
 # server-plugin-ore
 Generates ore veins
+
+## Add your own ore
+Other plugins can register ore blocks with this plugin by extending its configuration:
+´´´
+konstructs {
+  ...
+  org/konstructs/ore.ores {
+    // The config key is the block type of the ore, i.e.
+    // the block type of the ore is here my/domain/my-ore
+    my/domain/my-ore {
+      // This is the block type in which the ore can spawn
+      spawns-in = org/konstructs/stone
+
+      // This is the radius of the cube that needs to be of
+      // the spawns-in type for the ore to spawn. A radius of 3 is equal to a
+      // cube of 7x7x7 = 343 blocks (a block in the middle and 3 blocks in all directions)
+      radius = 3
+
+      // This is the number of generations the L-system will grow the vein
+      // Simplified, the number of blocks in the vein is 2^generations, so
+      // 5 generations is roughly equal to a vein of 32 blocks (can be both
+      // smaller and larger).
+      generations = 5
```

Comment:
Will the vein expand outside the `radius`? I assume so but it's not clear. Will it only replace hidden blocks of `spawns-in`, or is it possible for a vein to surface?